src/test/kotlin/com/sourcegraph/cody/agent/protocol/PositionTest.kt

```diff
@@ -1,7 +1,9 @@
 package com.sourcegraph.cody.agent.protocol
 
 import com.intellij.testFramework.fixtures.BasePlatformTestCase
+import org.junit.Ignore
 
+@Ignore
 class PositionTest : BasePlatformTestCase() {
```

Comment:
This is super annoying. We cannot have test like this.

Reply:
Why? What is the problem with this test?

Reply:
it is falky